### PR TITLE
Rename --no-json option to --table - Closes #380

### LINKED ISSUE #380: Rename --no-json option to --table
~~**This would be a breaking change.**~~
The same thing can be achieved with `--json false`.

Update: `--no-json` is automatically provided by Vorpal as an alias for `--json false`, but it would be better to officially support `--table` rather than `--no-json`.

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #380

### CODE REVIEW

File: src/utils/helpers.js [hunk 1]
```diff
@@ -70,12 +73,37 @@ export const createErrorHandler = prefix => ({ message }) => ({
 	error: `${prefix}: ${message}`,
 });
 
+const validateOutputFormatOptions = options => {
+	if (options.json && options.table) {
+		throw new Error('Cannot output both JSON and table.');
```
Comment (Tosch110) [inline src/utils/helpers.js]: In a validation function, prefer to throw the validation error

File: src/utils/helpers.js [hunk 2]
```diff
@@ -70,12 +73,37 @@ export const createErrorHandler = prefix => ({ message }) => ({
...
+	}
+	if (options.json === false && options.table === false) {
+		throw new Error('Must output either JSON or table.');
```
Comment (Tosch110) [inline src/utils/helpers.js]: as above

File: src/utils/options.js [hunk 3]
```diff
@@ -52,17 +49,20 @@ const passwordDescription = `Specifies a source for your secret password. Lisky
 const prettyDescription =
 	'Prints json in pretty format rather than condensed. Has no effect if json option is false. You can change the default behaviour in your config.json file.';
 
+const tableDescription =
+	'Sets output to table format (default). You can change the default behaviour in your config.json file.';
```
Comment (Tosch110) [inline src/utils/options.js]: Prefer `Prints output in table format (default)` as in prettyDescription. Same for `jsonDescription`

File: src/utils/helpers.js (hunk 1 above)
Comment (willclarktech) [reply src/utils/helpers.js]: The validation error is only in `0.3.0` so far. We can add this when `0.4.0` incorporates those changes.

**Pull request merged**